osmaxx-py/osmaxx/excerptexport/templates/excerptexport/templates/login.html

```diff
@@ -19,9 +19,27 @@
     <div>{{ form.password }}</div>
 </div>
 
-<input type="submit" value="login" />
-<input type="hidden" name="next" value="{{ next }}" />
+    <input type="submit" value="login" />
+    <input type="hidden" name="next" value="{{ next }}" />
 </form>
+<p>
+    {% trans 'or use Clavid:' %}<br />
+    <form method="post" action="{% url 'social:begin' 'clavid.ch' %}">{% csrf_token %}
+        <div>
+            <div><label for="openid_user">user name</label></div>
+            <div><input name="openid_user" type="text" required="required" />.clavid.ch</div>
```

Comment:
A placeholder text might be nice here. (Haven't ran the code though, so I don't know how it looks rendered.)